client/js/compute.js

```diff
@@ -0,0 +1,554 @@
+    },
+    {
+      name: '>=',
+      desc: 'returns true if x is greater than or equal to y',
+      sample: 'var a = ${x} >= ${y}',
+      call: function(v, x, y) { return v = x >= y }
+    },
+    {
+      name: '>',
+      desc: 'returns true if x is greater than y',
+      sample: 'var a = ${x} > ${y}',
+      call: function(v, x, y) { return v = x > y }
+    },
+    {
+      name: '&& (logical AND)',
+      desc: 'returns true if both x and y are true or returns the value of x or y if they are not boolean values',
+      sample: 'var a = ${x} >= ${y}',
+      call: function(v, x, y) { return v = x && y }
+    },
+    {
+      name: '|| (logical OR)',
+      desc: 'returns true if x or y or both are true or returns the value of x or y if they are not boolean values',
+      sample: 'var a = ${x} >= ${y}',
+      call: function(v, x, y) { return v = x || y }
+    },
+    {
+      name: '! (logical NOT)',
+      desc: 'returns true if the given value is false (or can be evaluated as false), otherwise returns false',
+      sample: 'var a = ! ${x}',
+      call: function(v, x) { return v = !x }
+    },
+    {
+      name: 'hypot',
+      desc: 'returns the square root of the sum of squares (hypotenuse) of its (up to 2) arguments',
+      sample: 'var a = hypot ${x} ${y}',
+      call: function(v, x, y) { return v = (y !== undefined) ? Math.hypot(x, y) : Math.hypot(x) }
+    },
+    {
+      name: 'max',
+      desc: 'returns the largest/highest of two numbers',
+      sample: 'var a = max ${x} ${y}',
+      call: function(v, x, y) { return v = Math.max(x, y) }
+    },
+    {
+      name: 'min',
+      desc: 'returns the smallest/lowest of two numbers',
+      sample: 'var a = min ${x} ${y}',
+      call: function(v, x, y) { return v = Math.min(x, y) }
+    },
+    {
+      name: 'pow',
+      desc: 'returns the base to the exponent power (x to y)',
+      sample: 'var a = pow ${x} ${y}',
+      call: function(v, x, y) { return v = Math.pow(x, y) }
+    },
+    {
+      name: 'sin',
+      desc: 'returns the sine of a number (in degrees)',
+      sample: 'var a = sin ${x}',
+      call: function(v, x) { return v = Math.sin(x * Math.PI/180) }
+    },
+    {
+      name: 'cos',
+      desc: 'returns the cosine of a number (in degrees)',
+      sample: 'var a = cos ${x}',
+      call: function(v, x) { return v = Math.cos(x * Math.PI/180) }
+    },
+    {
+	  name: 'tan',
+      desc: 'returns the tangent of a number (in degrees)',
+      sample: 'var a = tan ${x}',
+      call: function(v, x) { return v = Math.tan(x * Math.PI/180) }
+    },
+    {
+      name: 'asin',
+      desc: 'returns the arcsine (in degrees) of a number',
+      sample: 'var a = asin ${x}',
+      call: function(v, x) { return v = Math.asin(x) / Math.PI*180 }
+    },
+    {
+      name: 'acos',
+      desc: 'returns the arccosine (in degrees) of a number',
+      sample: 'var a = cos ${x}',
+      call: function(v, x) { return v = Math.acos(x) / Math.PI*180 }
+    },
```

Comment:
Line 149 should say:
sample: 'var a = acos ${x}',

(the a in front of cos is missing)